### PR TITLE
Feature/delete connection

### PR DESCRIPTION
**Pull request by blackinno**
- Delete connection.
- Display status `Activated` or `Deactivated`
- Twilio service.

Close https://github.com/mesg-foundation/aragon/issues/30

### CODE REVIEW

File: app/src/utils/eventsList.js [hunk 1]
```diff
@@ -38,6 +38,16 @@ export default {
         { name: 'text', label: 'Message', type: 'text', required: true }
       ]
     },
+    {
+      name: 'Twilio',
+      label: 'Send a message on SMS',
```
Comment (NicolasMahe) [inline app/src/utils/eventsList.js:43]:
```suggestion
      label: 'Send an SMS',
```

Comment (blackinno) [reply app/src/utils/eventsList.js]: Fix done.

File: app/src/utils/eventsList.js [hunk 2]
```diff
@@ -38,6 +38,16 @@ export default {
         { name: 'text', label: 'Message', type: 'text', required: true }
       ]
     },
+    {
+      name: 'Twilio',
+      label: 'Send an SMS',
```
Comment (NicolasMahe) [inline app/src/utils/eventsList.js:43]:
```suggestion
      label: 'Send a SMS',
```

**Comment by NicolasMahe** [comment]
@blackinno good job. I let you merge when you want

**Pull request merged**